### PR TITLE
MCAR-19: Expose feature set flags to hide Devices configuration in Call Recoding

### PR DESCRIPTION
**Pull request by ramandeepromana**

### CODE REVIEW

File: src/js/lib/monster.js [hunk 1]
```diff
@@ -157,6 +157,16 @@ define(function(require) {
 						eventRouting: true
 					}
 				}
+			},
+			callRecording: {
+				28: {
+					configuration: {
+						manage: true,
```
Comment (guillegr123) [inline src/js/lib/monster.js:164]: Is this value (`callRecording.configuration.manage`) going to be read in any app? I don't see it read at [MCAR-19](https://github.com/2600hz/monster-ui-call-recording/pull/30/files)

Comment (ramandeepromana) [reply src/js/lib/monster.js:164]: Oh no and I added that as Devices section falls under Configuration 🤔 

My thinking was something along the lines of `callRecording: {` similar to `smartpbx: {` with `28: {` under both of them

& `configuration: {` section under `callRecording` with  option to `manage` as we will be working on that, followed by `devices: { manage:` as the falls under Configuration

Comment (guillegr123) [reply src/js/lib/monster.js:164]: Oh, OK, got it 👍

**Review by guillegr123 — APPROVED** [review]
Looks good to me!

**Pull request merged**